recipes/setuptools/bld.bat

```diff
@@ -0,0 +1,8 @@
+python setup.py install
```

Comment:
feel free to take this one on @jakirkham.  I'm pretty tied up trying to
wrap up the conda-build release

On Fri, May 6, 2016 at 12:45 AM jakirkham <notifications@github.com> wrote:

> In recipes/setuptools/bld.bat
> <https://github.com/conda-forge/staged-recipes/pull/551#discussion_r62290687>
> :
>
> > @@ -0,0 +1,8 @@
> > +python setup.py install
>
> Nope. Seems like this file doesn't help. Weird that it has the same as the
> script we are recommended to download.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/conda-forge/staged-recipes/pull/551/files/939cbf67ac6b4f0c556b2ab8328dd774c990fc8d#r62290687>
>